src/app/app-routing.module.ts

```diff
@@ -31,8 +31,7 @@ const routes: Routes = [
   },
   {
     path: 'test-results/:resource/:subjectIdentifier',
-    component: TestResultsComponent,
-    canActivate: [AuthGuardService]
```

Comment:
Why did you remove this?

Reply:
You can read the commit message for https://github.com/release-engineering/estuary/pull/111/commits/4ec58d7b6bc34bdf1abddb01304ff63d2c0eb8c6.

Reply:
Ok, sorry. Didn't see that :)
But we discussed about that over IRC while you were on PTO.
With many folks on the team and with @ralphbean too. This is not just Greenwave, but also ResultsDB, WaiverDB, Freshmaker... so we said we might want to discuss it a bit more and take a decision that is global across all these services and Estuary.

Reply:
I wrote that in the last comment in the JIRA card: https://projects.engineering.redhat.com/browse/FACTORY-4877

Reply:
@gnaponie sorry. I missed that comment. It requires auth now only because I arbitrarily made that decision when I added the test results page. There was never a requirement for it as far as I know.

If you still don't feel comfortable with that, I'll drop that commit from this PR.

Reply:
I'm kinda of your same idea. There's no reason in showing the data in one place and not showing them in another. But there were other opinions around the team. And mostly we should have it consistent across all services. So we thought maybe we just have to talk a bit about it. Maybe can we drop the commit, and file a spike card to decide what to do about it? We can have a short meeting or just a document. We can include the team and the pipeline experts (Ralph, Mike, someone else?).